modules/helfi_paragraphs_org_chart/src/OrgChartImporter.php

```diff
@@ -51,14 +58,24 @@ private function getUri(string $langcode, string $start, int $depth): string {
    *   The data.
    */
   public function fetch(string $langcode, string $start, int $depth) : array {
+    // Return mock response if the use mock feature is enabled.
+    if ($this->featureManager->isEnabled(FeatureManager::USE_MOCK_RESPONSES)) {
+      $data = file_get_contents(__DIR__ . "/../tests/fixtures/org-chart-$depth.json");
+      return Utils::jsonDecode($data, assoc: TRUE);
+    }
+
     try {
       $data = $this->client->request('GET', $this->getUri($langcode, $start, $depth))
         ->getBody()
         ->getContents();
       $chart = Utils::jsonDecode($data, assoc: TRUE);
     }
     catch (GuzzleException | InvalidArgumentException) {
-      return [];
+      return [
```

Comment:
`OrgChartLazyBuilder` has:

```php
    $data = $this->storage->fetch($langcode, $start, $depth);
    return [
      ...
      '#cache' => [
        // Cache for 1 day on successful requests.
        'max-age' => empty($data) ? 60 : 86400,
      ],
      ...
    ];
```

Which breaks if this no longer returns empty array on failure. I propose moving the error messages to the lazy builder instead.

Reply:
The error messages were moved to templates and the error message handling was moved to the lazy builder.